### PR TITLE
Added best practices section for Collect

### LINKED ISSUE #92: Tips for location
Different phones have different GPSs.

Accuracy and battery can be controlled in Android settings - https://support.google.com/nexus/answer/3467281?hl=en

### LINKED ISSUE #210: Create Android desktop shortcut to launch specific form
https://forum.opendatakit.org/t/opening-a-form-automatically-when-collect-starts/1649 gives basic instructions. They should be verified with latest Android versions. Collect now supports 4.1 and above so I don't think we need instructions for older devices.

http://www.samsung.com/uk/support/skp/faq/1057750 has some good background on Widgets.

### PR DESCRIPTION
**Pull request by ankita240796**
addresses #106 
closes #210 
closes #92 

## What is included in this PR?
- Added information on creating shortcut for a form on home screen.
- Added tips about location services.

### CODE REVIEW

**Comment by adammichaelwood** [comment]
Please re-do the images. This display font is not acceptable.

**Comment by adammichaelwood** [comment]
The content seems otherwise fine.

**Comment by ankita240796** [comment]
@adammichaelwood I updated the images.

File: collect-best-practices.rst [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+   4. Information about adding widgets will be displayed. You can then follow it or try it to find the :guilabel:`Widgets` button.
+
+   .. image:: /img/collect-best-practices/help-describe.png
+      :alt: Image showing guidelines on adding a widget.
+      :class: device-screen-vertical
+
+.. _location-tips:
+
+Tips on Location
```
Comment (adammichaelwood) [inline collect-best-practices.rst]: Each of these individual tips needs its own section heading

```
Location tips
------------------

Enable location providers
~~~~~~~~~~~~~~~~~~~~~~~
.
.

Location mode     (or something like that)
~~~~~~~~~~~~~~

.
.
.
Improving location accuracy
~~~~~~~~~~~~~~~~~~~~~~~~
.
.
.
```

File: collect-best-practices.rst [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+Creating Android desktop shortcut to launch specific form
+------------------------------------------------------------
+
+1. Click on :guilabel:`Apps` button at the bottom of the screen.
+
+.. image:: /img/collect-best-practices/apps.png
+   :alt: Image showing Apps button.
+   :class: device-screen-vertical
+
+2. Touch and hold on an empty space on the Home screen and then click on :guilabel:`Widgets` button at the bottom of the screen.
```
Comment (adammichaelwood) [inline collect-best-practices.rst]:
```
:gesture:`Touch and hold`
```

fix this throughout -- also:

```
:gesture:`drag and drop`
:gesture:`tap`
etc...
```

**Comment by ankita240796** [comment]
@adammichaelwood I made the requested changes.

**Pull request merged**